qiita_db/analysis.py

```diff
@@ -753,7 +753,7 @@ def _add_file(self, filename, filetype, data_type=None, conn_handler=None):
                "checksum, checksum_algorithm_id, data_directory_id) VALUES "
                "(%s, %s, %s, %s, %s) RETURNING filepath_id")
         # magic number 1 is for crc32 checksum algorithm
-        fpid = conn_handler.execute_fetchone(sql, (fullpath, fptypeid,
+        fpid = conn_handler.execute_fetchone(sql, (filename, fptypeid,
```

Comment:
I can for the filepath insertion, but I still need it for the linking of analysis to the filepath.